test/factories/CommandFactoryTest.ts

```diff
@@ -3,47 +3,70 @@ import { expect } from "chai";
 import CommandFactory from "../../src/factories/CommandFactory";
 // @ts-ignore - TS does not like MockCommand not living in src/
 import MockCommand from "../MockCommand";
+import MockCommandWithAlias from "../MockCommandWithAlias";
 
```

Comment:
This is weird. The linter was complaining about indentation problems "Expected 2 tabs, found 8 spaces". Autofixing would change the entire file, and it was rendering ok on my editor, so I left it like it was. However, when I pushed it to GitHub, the rendering was pretty weird, so I pushed another commit with the autofix. Let me know if this was a bad call.

Reply:
Yeah, so your editor will show four spaces and a tab being the same width, but GitHub will show them differently.